src/collections/components/ChipLabelsList/ChipLabelsList.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { Button, Collapse } from '@mui/material';
+import LabelOutlinedIcon from '@mui/icons-material/LabelOutlined';
+import ExpandLess from '@mui/icons-material/ExpandLess';
+import ExpandMore from '@mui/icons-material/ExpandMore';
+
+import { Collection } from '../../../api/generatedTypes';
+import { StyledChipComponent } from '../../../_shared/styled';
+
+interface ChipLabelsListProps {
+  /**
+   * An object with everything collection-related in it.
+   * Except for stories.
+   */
+  collection: Omit<Collection, 'stories'>;
+}
+
+/**
+ * A custom styled chip label list component to display labels and expand more/less button.
+ *
+ * @param props
+ */
+export const ChipLabelsList: React.FC<ChipLabelsListProps> = (props) => {
+  const { collection } = props;
+
+  // expand more/less button state
+  const [expandButtonToggled, setExpandButtonToggle] = useState(true);
+
+  // onMouseDown event is used when pressing the mouse button on the expand button.
+  // this event is needed in order to fire logic as soon as the mouse is pressed, instead of waiting for the full click.
+  const handleMouseDown = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
```

Comment:
can pass a flag and make this optional as its only needed for the `CollectionListCard` component. `CollectionInfo` does not have a clickable card.